wdl-analysis/src/queue.rs

```diff
@@ -700,3 +730,16 @@ where
         (index, document)
     }
 }
+
+/// Formats the panic payload for display.
+pub(crate) fn format_panic_payload(payload: &Box<dyn std::any::Any + Send>) -> String {
+    if let Some(s) = payload.downcast_ref::<&str>() {
+        s.to_string()
+    } else if let Some(s) = payload.downcast_ref::<String>() {
+        s.clone()
+    } else if let Some(loc) = payload.downcast_ref::<&Location<'static>>() {
+        format!("panic occurred at {}", loc)
```

Comment:
I believe (but definitely could be wrong) that the panic payload would never be a `&Location` and there isn't a mechanism to get a location from within an already unwound stack (i.e. from `panic::catch_unwind`).

The only way the panic payload would be a `&Location` is if someone used `std::panic::panic_any` with a `&'static Location<'static>`, which would be odd (but not impossible); the normal `panic!` machinery, as far as I'm aware, currently only delivers a `&'static str` or `String` payload to the unwind. The internal `&Location` is only available to the panic hook, which is pre-unwind.

So I think this `else if` clause can be removed, personally.